docs/linters.md

```diff
@@ -0,0 +1,210 @@
+# Linters
```

Comment:
yep sgtm, I was actually kinda thinking even as I wrote it that it could be a fine section at the bottom really.